gpgwrapper/gpgwrapper.go

```diff
@@ -4,12 +4,20 @@ package gpgwrapper
 
 import (
 	"errors"
+	"fmt"
+	"io"
 	"os/exec"
 	"regexp"
 )
 
 const GpgPath = "gpg"
 
+var ErrNoVersionStringFound = errors.New("version string not found in GPG output")
+
+func ErrProblemExecutingGPG(arguments ...string) error {
```

Comment:
I think it would be incredibly helpful if the error message *also* contained the contents of `stdin` and `stderr` otherwise you can't see what GnuPG complained about when it bombed out